app/presenters/mixins/last_operation_helper.rb

```diff
@@ -0,0 +1,21 @@
+module VCAP::CloudController::Presenters::Mixins
+  module LastOperationHelper
+    def last_operation(resource)
+      last_operation = resource.try(:last_operation)
```

Comment:
If I understand correctly `resource.try` is safe in the case `resource` is `nil`. Can that really happen?

Reply:
oh, just checked and it also swallows no_method error so I guess that's why is here